modules/react/tooltip/spec/Tooltip.spec.tsx

```diff
@@ -59,6 +62,25 @@ describe('Tooltip', () => {
     });
   });
 
+  describe('when "showDelay" is passed in', () => {
+    jest.useFakeTimers();
+    it('should render the tooltip after the delay', () => {
+      render(
+        <Tooltip type="describe" title="Delayed Tooltip Text" showDelay={1000}>
+          <span>Test Text</span>
+        </Tooltip>
+      );
+
+      fireEvent.mouseEnter(screen.getByText('Test Text')); // triggers the tooltip
+
+      expect(screen.queryByText('Delayed Tooltip Text')).toBeNull(); // tooltip is not shown before the delay
+
+      jest.advanceTimersByTime(1000); // advance the timer by the amount of delay time
```

Comment:
This is validating the delay is at least 1000ms and more than 0, but not that the override works correctly. The following will fix that:

```suggestion

      jest.advanceTimersByTime(300); // advance the timer by the default delay time
      expect(screen.queryByText('Delayed Tooltip Text')).toBeNull(); // tooltip is not shown before the configured delay

      jest.advanceTimersByTime(700); // advance the timer by the amount of delay time
```